wiki/Beatmaps/en.md

```diff
@@ -1,78 +1,67 @@
----
-outdated: true
----
-
 # Beatmaps
 
-*Not to be confused with: [Difficulties](/wiki/Difficulties).*
+*Not to be confused with [Difficulties](/wiki/Difficulties).*
 
-Beatmaps are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games). In their most basic form, a beatmap have a song and various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music. They can also have a background image, video, and/or [storyboard](/wiki/storyboard).
+A beatmap is a set of difficulties in [osu!](/wiki/osu!_glossary). Beatmaps consist of a song, a background image and multiple [hit objects](/wiki/hit_objects) in sync with the beat. They can also include a video, and/or [storyboard](/wiki/storyboards).
 
-Beatmaps can be uploaded by the [BSS](/wiki/BSS) (Beatmap Submission System) and are then run through an approval process called [modding](/wiki/modding) in order to maintain the high standards of playability.
+Beatmaps are shared through a system called [BSS](/wiki/BSS) (Beatmap Submission System). After beatmaps have been shared they will show up on the [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) page on osu! web where they can be downloaded and favorited.
 
-You can find beatmaps in the [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) page on osu! web.
+You can also find beatmaps in [beatmap packs](https://osu.ppy.sh/beatmaps/packs) or through osu!direct.<!--TODO: Add link once osu!supporter article has been written-->
 
-## Ranked Statuses
+## Beatmap Categories
 
-Both ranked and approved beatmaps requires approval from two [Beatmap Nominators](/wiki/Beatmap_Nominators). They can influence the stats on a player's profile which is what makes them different from a [pending](#pending) or [graveyarded](#graveyard) beatmap.
+Beatmaps are split into 5 different categories depending on their recent activity, nomination status and/or favorites. The categories are:
 
-### Ranked
+- Graveyard
+- Work in Progress/Pending
+- Qualified
+- Ranked
+- Loved
+
+### Graveyard
+
+Graveyarded beatmaps are beatmaps that have been abandoned by their creator. Beatmaps are moved to the Graveyard from the [pending](#pending) category automatically once they haven't been updated for 4 weeks. Graveyarded beatmaps do not have leaderboards and will not affect user statistics, but can still be downloaded and played like normally.
+
+Once a beatmap is graveyarded it can be revived (put back into pending) by updating the beatmap through the [BSS](/wiki/BSS) (Beatmap Submission System)
```

Comment:
```suggestion
Once a beatmap is graveyarded, it can be revived (i.e. being it back to the pending category). This can be done by updating the beatmap via [BSS](/wiki/BSS) (Beatmap Submission System).
```